utilities/src/main/java/io/syndesis/qe/bdd/CommonSteps.java

```diff
@@ -136,4 +164,79 @@ public void deployFTPServier() {
     public void cleanFTPServier() {
         FtpTemplate.cleanUp();
     }
+
+    @Given("^db to db \"([^\"]*)\" integration with period (\\d+) ms$")
```

Comment:
I don't think that this whole metod should end up in `Common` class. @tplevko could you pls make sure that we follow the same approach as rest of `rest-tests`.

Reply:
@dsimansk this is perhaps something else, what I had on mind. My intention was following: there are some UI features that requires published integration. The quickest way to create one is via rest way. 
There is a problem to add rest-test as dependency of ui-tests, so that's why I was trying to put here in utilities. 
After discussion with @tplevko we found solution in creating another step class in utilities (since CommonSteps are really much more common than this.)